### DISCUSSION TITLE
How to show YAML in pop-up preview?

### DISCUSSION THREAD

**Discussion by tjoflong** [Richardsl/heatmap-calendar-obsidian#94] [Q&A]
In the documentation there is an example (the red exercise heatmap) where I can see the YAML of the underlying note in the popup preview window.

How can I do that? 

For me the popup window shows no YAML info.

**Comment by occultliquidity** [comment]
Download the plugin called Hover editor. It allows you to see all the YAML front matter 
![Screenshot 2024-12-14 at 15 29 42](https://github.com/user-attachments/assets/00eb6e7e-9233-418b-b35c-9c7a9c0c6a23)
